synapse/lib/trigger.py

```diff
@@ -359,8 +359,12 @@ async def _execute(self, node, vars=None, view=None):
             'user': self.user.iden,
         }
 
-        if vars is not None:
-            opts['vars'] = vars
+        if vars is None:
+            vars = {}
+        else:
+            vars = vars.copy()
+        vars['auto'] = {'iden': self.iden, 'type': 'trigger'}
```

Comment:
Does this need to be a setdefault() just in case?

Reply:
Triggers do not do capture of user variables. Their vars are populated prior to execution; but I do need to move this vars population up one level upon review.